cpp/demo/poisson-matrix-free/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# This file was generated by running
+#
+#     python cmake/scripts/generate-cmakefiles from dolfinx/cpp
+#
+cmake_minimum_required(VERSION 3.16)
+
+set(PROJECT_NAME matrix_free)
```

Comment:
Was this file generated by our Python script that creates CMake script? The project name points to it being added manually. Could cause a problem later when re-running the script.